src/Operators/areas_and_volumes.jl

```diff
@@ -0,0 +1,42 @@
+####
+#### Grid spacings
+####
+
+@inline Δx(i, j, k, grid) = grid.Δx
+@inline Δy(i, j, k, grid) = grid.Δy
+
+@inline ΔzC(i, j, k, grid::RegularCartesianGrid) = grid.Δz
+# @inline ΔzC(i, j, k, grid::VerticallyStretchedCartesianGrid) = @inbounds grid.ΔzC[k]
```

Comment:
I understand this is a placeholder, but we probably want to dispatch on whether `zC` is provided as a function or array for the vertically stretched grid.

Reply:
Hmmm, in designing the vertically stretched grid I've assumed that `zF` and `zC` will be arrays because I don't see the benefit of allowing it to be a function (extra complication, but 

Users can supply either an array or a function to `VerticallyStretchedCartesianGrid` to specify `zF` or `zC` (or `ΔzC` or `ΔzF`) so if a function is passed, then it will be used to construct a 1D array.

We can discuss this in an upcoming PR about the vertically stretched grid struct.

Reply:
I think implementing dispatch at the lowest level to support flexible user input *could* be simpler in terms of code logic than wrangling user-inputs into a desired form in the constructor (if the user supplies a function, it is used directly in the algorithm, no need to check the constructor to understand translation steps). And there are probably performance benefits for some cases (simple stretching factors), and not for others. That is left to user discretion.

For 3D non-uniform grids, the benefit of functions may become more clear. Anticipating that, I think it could be nice for code logic to be uniform across grid types.

Since supporting functions is trivial (one extra line in `Operators`; no extra lines in constructor if we already wish to support functions), it may be worth the effort?